api/src/pcapi/routes/backoffice/templates/finance/incidents/get/general_commercial_gesture.html

```diff
@@ -35,7 +35,13 @@ <h6 class="card-subtitle text-muted">ID : {{ incident.id }}</h6>
         <div class="d-flex justify-content-start">
           <div class="d-flex flex-column pe-5">
             <p class="mb-1">
-              <span class="fw-bold">Lieu porteur de l'offre :</span> {{ links.build_venue_name_to_details_link(incident.venue) }}
+              <span class="fw-bold">
+                {% if is_feature_active("WIP_ENABLE_OFFER_ADDRESS") %}
+                  Partenaire culturel :
```

Comment:
Choix ou oubli ? Dans le diff en-dessous (general_overpayment.html) on garde « porteur de l'offre » dans les deux cas.
```suggestion
                  Partenaire culturel porteur de l'offre :
```